### ISSUE TITLE
Update disaggregation accordions

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3081]
The drop-down boxes under the ‘Sub-categories’ heading page do not indicate to the user that they are expandable or collapsible. Ensure that the state of the expandable and collapsible content is identified. For example, the elements should be read as expandable or collapsible. Consider providing ariaexpanded="false" and change the value of the aria-expanded attribute as per the action.

Suggest investigating `aria` tag and finding a way to signal whether the accordion is open or closed.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Sprint 12](https://trello.com/c/HZJO8O4J/576-accessibility-sprint-12)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [ ] Windows: IE11
- [ ] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides is this done?

**Comment by SavvasStephanides** [comment]
Pull requests:

`open-sdg`: https://github.com/open-sdg/open-sdg/pull/200
`sdg-indicators`: https://github.com/ONSdigital/sdg-indicators/pull/3163

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides does the branch need to be rebuilt now that develop has been updates with 0.6.1?

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Update disaggregation accordions](https://trello.com/c/7DHC7GSx/605-update-disaggregation-accordions)

**Comment by Norric1Admin** [comment]
@LucyGwilliamAdmin is aligning `a11y-ariaexpanded-fix` with the new `develop` branch, which contains the open-sdg update. This has been complicated by a problem with GitHub's 'resolve conflicts' feature.

**Comment by LucyGwilliamAdmin** [comment]
@Norric1Admin @SavvasStephanides  I think this can be tested [here](https://sdg.mango-solutions.com/a11y-ariaexpanded-fix/) now, which is probably better since once we know it's working we can just merge into open-sdg and then we don't need to overwrite the file

**Comment by LucyGwilliamAdmin** [comment]
> Pull requests:
> 
> `open-sdg`: [open-sdg/open-sdg#200](https://github.com/open-sdg/open-sdg/pull/200)
> `sdg-indicators`: #3163
> 
> Test URL: https://sdg.mango-solutions.com/a11y-ariaexpanded-fix/
> 
> **NOTE:** The `open-sdg` pull request will need to be approved first in order for `sdg-indicators` to work properly.

@SavvasStephanides was there something in particular that needed to come from open-sdg? The feature branch is now using v0.6.1 but I'm still having the same issues as before

**Comment by Norric1Admin** [comment]
@SavvasStephanides Trying to link directly to `https://github.com/ONSdigital/sdg-indicators/issues/3081` instead of the open-sdg@0.6.1 update which included it, in case I missed something.

**Comment by Norric1Admin** [comment]
@SavvasStephanides The only problem with this is (I think) the ARIA tab reading 'space' as "space", rather than the expected "Checkbox checked/unchecked".
Try this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

**Comment by LucyGwilliamAdmin** [comment]
@SavvasStephanides do you know if this was fixed? Don't have a Samsung but have Android. Tested on Android Chrome and Android browser and seems to work on those

**Comment by AnnCorp** [comment]
Resource focussed on achieving AA, so won't be actioning AAA fixes currently.
